content/post/2019-01-20-what-is-production-grade.markdown

```diff
@@ -0,0 +1,270 @@
+---
+title: "What Is Production Grade?"
+date: 2019-01-20T15:35:03-08:00
+---
+
+I recently decided to try the Caddy webserver project's implementation
+of a webserver written in Golang.  It had many interesting things
+going for it, including automagic certificate renewal via ACME and
+Let's Encrypt, "plugins" written in a sane language, dramatically
+simplified configuration, and perhaps in one of the most boastful
+statements I've seen of any software recently, it claims to be
```

Comment:
English pedant hat: When a list contains commas, the list separator is promoted to a semicolon.